qa/tasks/deepsea.py

```diff
@@ -406,6 +406,7 @@ def _populate_deepsea_context(self):
                 )
         deepsea_ctx['repositories'] = self.config.get("repositories", None)
         deepsea_ctx['rgw_ssl'] = self.config.get('rgw_ssl', False)
+        deepsea_ctx['validation_tests_already_run'] = []
```

Comment:
Can we use shorter label here, like 'test_run_list' or 'tests_run', or 'run_status'?

Reply:
I prefer the longer name - to me, it makes the intended purpose of this ctx value immediately clear. My memory is so bad ... with a more cryptic name I would forget the meaning and become confused.